nym-api/nym-api-requests/src/models/described.rs

```diff
@@ -309,12 +309,26 @@ impl From<Authenticator> for AuthenticatorDetails {
         }
     }
 }
+
+fn default_tunnel_port() -> u16 {
+    51822
+}
+
+fn default_metadata_port() -> u16 {
+    51830
+}
+
```

Comment:
@neacsu I needed to add these, because the NS API had trouble deserialising old `WireguardDetails` structs